<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coulr</string>
  </property>
  <property name="windowIcon">
   <iconset theme="coulr">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget1">
      <property name="minimumSize">
       <size>
        <width>172</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string>Render</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QWidget" name="color_widget" native="true"/>
        </item>
        <item>
         <widget class="QLineEdit" name="html_color">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>Palette</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0">
         <widget class="QWidget" name="color3" native="true"/>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="color4" native="true"/>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="color1" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="color2" native="true"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="color2_html"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="color1_html"/>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="color3_html"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="color4_html"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>RGB</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>G</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="red">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="green">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="blue">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSlider" name="horizontalSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>HSL</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>H</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="lightness">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="saturation">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>L</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="hue">
          <property name="maximum">
           <number>359</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSlider" name="horizontalSlider_4">
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSlider" name="horizontalSlider_5">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSlider" name="horizontalSlider_6">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>HSV</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QSpinBox" name="hue2">
          <property name="maximum">
           <number>359</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>H</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="saturation2">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="value">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSlider" name="horizontalSlider_7">
          <property name="maximum">
           <number>359</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSlider" name="horizontalSlider_8">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSlider" name="horizontalSlider_9">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPick_color">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/picker.png</normaloff>icons/picker.png</iconset>
   </property>
   <property name="text">
    <string>Pick color</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget1</tabstop>
  <tabstop>red</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>green</tabstop>
  <tabstop>horizontalSlider_2</tabstop>
  <tabstop>blue</tabstop>
  <tabstop>horizontalSlider_3</tabstop>
  <tabstop>html_color</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lightness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hue2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturation2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hue2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturation2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
